client/src/app/site/pages/meetings/modules/projector/modules/slides/components/current-speaker-chyron-slide/components/current-speaker-chyron-slide.component.scss

```diff
@@ -1,33 +1,61 @@
 @import 'src/assets/styles/fonts.scss';
 
-#chyron {
+#chyron,
+#chyron_new {
     position: absolute;
     left: 50px;
     bottom: 20px;
     right: 0;
     z-index: 10;
-    height: 80px;
-    font-size: 32px;
+    margin-right: 50px;
```

Comment:
please use right: 50px instead of margin-right

Reply:
done.